ls/server/src/test/java/org/opencds/cqf/cql/ls/server/command/DebugCqlCommandContributionTest.java

```diff
@@ -0,0 +1,156 @@
+                contribution.getCommands().toArray()[0]);
+    }
+
+    @Test
+    void withOnlyLibrary() {
+        ExecuteCommandParams params = new ExecuteCommandParams();
+
+        String libraryPath = normalizePath(
+                "file://" + System.getProperty("user.dir") + "/src/test/resources/org/opencds/cqf/cql/ls/server/");
+
+        params.setCommand("org.opencds.cqf.cql.ls.plugin.debug.startDebugSession");
+        params.setArguments(Arrays.asList("cql", "-fv=R4", "-m=FHIR", "-ln=One", "-lu=" + libraryPath).stream()
+                .map(str -> "\"" + str + "\"")
+                .map(JsonParser::parseString)
+                .collect(Collectors.toList()));
+        Object result = contribution.executeCommand(params).join();
+        assertInstanceOf(String.class, result);
+        assertTrue(((String) result).trim().endsWith("One=1"));
+    }
+
+    @Test
+    void withTerminology() {
+        ExecuteCommandParams params = new ExecuteCommandParams();
+
+        String libraryPath = normalizePath(
+                "file://" + System.getProperty("user.dir") + "/src/test/resources/org/opencds/cqf/cql/ls/server/");
+
+        params.setCommand("org.opencds.cqf.cql.ls.plugin.debug.startDebugSession");
+        params.setArguments(
+                Arrays.asList("cql", "-fv=R4", "-m=FHIR", "-ln=One", "-lu=" + libraryPath, "-t=" + libraryPath).stream()
+                        .map(str -> "\"" + str + "\"")
+                        .map(JsonParser::parseString)
+                        .collect(Collectors.toList()));
+        Object result = contribution.executeCommand(params).join();
+        assertInstanceOf(String.class, result);
+        assertTrue(((String) result).trim().endsWith("One=1"));
+    }
+
+    @Test
+    void withFileModel() {
+        ExecuteCommandParams params = new ExecuteCommandParams();
+
+        String modelPath = normalizePath(
+                "file://" + System.getProperty("user.dir") + "/src/test/resources/org/opencds/cqf/cql/ls/server/");
+        String libraryPath = normalizePath(
+                "file://" + System.getProperty("user.dir") + "/src/test/resources/org/opencds/cqf/cql/ls/server/");
+
+        params.setCommand("org.opencds.cqf.cql.ls.plugin.debug.startDebugSession");
+        params.setArguments(Arrays.asList(
+                        "cql",
+                        "-fv=R4",
+                        "-m=FHIR",
+                        "-mu=" + modelPath,
+                        "-ln=One",
+                        "-lu=" + libraryPath,
+                        "-t=" + libraryPath)
+                .stream()
+                .map(str -> "\"" + str + "\"")
+                .map(JsonParser::parseString)
+                .collect(Collectors.toList()));
+        Object result = contribution.executeCommand(params).join();
+        assertInstanceOf(String.class, result);
+        assertTrue(((String) result).trim().endsWith("One=1"));
+    }
+
+    //    @Test
+    //    void withRemoteModel() {
```

Comment:
This requires us to stand up a dummy HTTP server which is a rabbit hole I'm not gonna go down now, but this test does work locally with a FHIR server running, so it still has some value.